install_config/install/host_preparation.adoc

```diff
@@ -710,6 +710,39 @@ An easy way to distribute your SSH keys is by using a `bash` loop:
 
 Modify the host names in the above command according to your configuration.
 
+
+[[setting-global-proxy]]
+== Setting Global Proxy Values
+
+The {product-title} installer uses the proxy settings
+in the *_/etc/environment _* file.
+
+List the following domain suffixes and IP addresses in the *_/etc/environment_* file in the `no_proxy` parameter:
+
+* Master and node hostnames (domain suffix)
+* Other internal hostnames (domain suffix)
+* Etcd IP addresses (must be IP addresses, not host names as etcd access is done by IP address)
+* Docker registry IP address
+* Kubernetes IP address, by default 172.30.0.1. Must be the value set in the
+xref:../../install_config/install/advanced_install.adoc#advanced-install-networking-variables-table[openshift_portal_net] parameter in the 
+Ansible inventory file, by default *_/etc/ansible/hosts_*.
+* Kubernetes internal domain suffix: `cluster.local`. 
+* Kubernetes internal domain suffix: `.svc`
+
+For example:
+
+----
+no_proxy=.internal.example.com,10.x.x.1,10.x.x.2,10.x.x.3,172.30.0.1,.cluster.local,.svc,localhost,127.0.0.1
```

Comment:
I'd be worried that this `10.x.x.1` would lead the admin to believe that `x.x` somehow has some special meaning like it's a wildcard, but it doesn't. I'd just say `10.0.0.1,10.0.0.2,10.0.0.3` instead.